src/Domain/CreateInitialElement.cpp

```diff
@@ -33,109 +111,70 @@ Element<VolumeDim> create_initial_element(
   const auto& neighbors_of_block = block.neighbors();
   const auto segment_ids = element_id.segment_ids();
 
-  // Declare two helper lambdas for setting the neighbors of an element
```

Comment:
This lambda is completely rewritten and reviewers should ignore the interleaved diffs